uast/uast.go

```diff
@@ -377,6 +377,12 @@ const (
 	// annotations.
 	OtherLiteral
 
+	// MapKeyValue is the expression pairing a map key and a value usually on MapLiteral expressions. It must
+	// have both a MapKey and a MapValue children (e.g. {"key": "value", "otherkey": "otherval"} in Python).
+	MapKeyValue
```

Comment:
Setting the precedent of naming things by enumerating their contents is dangerous.  May we can call this MapEntry, MapEntryLiteral instead.